packages/xod-cli/README.md

```diff
@@ -85,6 +87,68 @@ EXAMPLES
 
 _See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.1.0/src/commands/autocomplete/index.ts)_
 
+## `xodc boards [options]`
+
+show available boards
+
+```
+USAGE
+  $ xodc boards [options]
+
+OPTIONS
+  -V, --version         show CLI version
+  -h, --help            show CLI help
+  -q, --quiet           do not log messages other than errors
+  -w, --workspace=path  [default: ~/xod] use the workspace specified, defaults to $HOME/xod
+  --available           show available boards
+  --installed           show only installed boards
+```
+
+_See code: [src/commands/boards.js](https://github.com/xodio/xod/blob/master/packages/xod-cli/src/commands/boards.js)_
+
+## `xodc compile [options] [entrypoint]`
+
+compiles (verifies) a XOD program
+
+```
+USAGE
+  $ xodc compile [options] [entrypoint]
+
+ARGUMENTS
+  ENTRYPOINT
+      Project and/or patch to operate on. The project should point to a file or
+      directory on the file system. The patch may either point to file system or
+      be a XOD patch path. If either is omitted, it is inferred from the current
+      working directory or another argument. Examples:
+
+         * ./path/to/proj.xodball main      # xodball + patch name
+         * ./path/to/proj/main/patch.xodp   # just full path to a patch
+         * main                             # a patch in the current project
+
+OPTIONS
+  -V, --version         show CLI version
+  -b, --board=fqbn      (required) target board identifier
+  -h, --help            show CLI help
+
+  -o, --output=path     save output binary at the path specified; could be a file path or directory; the same directory
```

Comment:
```suggestion
  -o, --output=path     save output binary at the path specified; can be a file path or directory; the same directory
```

Reply:
“the same directory” is not clear to me

Reply:
The string is copied from issue :)
Actually, there is a bug. I always use this path as the directory path and copy here ino, elf and hex/bin. Do we really need the option to save only firmware here? If yes, what logic here? If the path exists and is a directory, then output all files to the directory, or if a path not exists/file, then save firmware to the path?

Reply:
> The string is copied from issue :)

Regardless I made the issue the sentence is not clear to me :rofl:
To be honest, my bad, I missed the next line: “the same directory is used for intermediate build artifacts; defaults to `cwd`” is fine.


> Actually, there is a bug. I always use this path as the directory path and copy here ino, elf and hex/bin.

It is OK, I think. When I wrote about the path here I though about putting binaries for multiple platforms in a single directory. No longer actual. So, I suggest rewording to reflect the current state:

```
-o, --output=dir   save the result binary to the directory; the same directory is used
                   for intermediate build artifacts; defaults to `cwd`
```